frontend/src/app/apps/error.tsx

```diff
@@ -11,7 +15,7 @@ export default function Error({ message, reset }: any) {
 				className={styles.tryAgainButton}
 				onClick={
 					// Attempt to recover by trying to re-render the segment
-					() => reset()
+					() => router.replace(pathName)
```

Comment:
What does this do?

Reply:
Refreshes the page instead of using the reset function. Refreshing now activates middleware and will redirect to login if they aren't logged in. If they are logged in then it accomplishes the same function as the reset function by attempting to fetch the applications again through refresh. Helps in scenarios like clicking the back button after logging out, etc.